saleor/checkout/utils.py

```diff
@@ -542,15 +544,6 @@ def recalculate_checkout_discount(
                 if voucher.translated.name != voucher.name
                 else ""
             )
-            checkout.save(
-                update_fields=[
-                    "translated_discount_name",
-                    "discount_amount",
-                    "discount_name",
-                    "currency",
-                    "last_change",
-                ]
-            )
```

Comment:
Are we sure that we should remove this save? Whole `else` statements change `checkout` objects and don't save them.

Reply:
It was moved to: https://github.com/saleor/saleor/pull/9242/files#diff-3a4d5975bee96cf58c1c7f362360641d5cf6030a1817e0b14ea92561d98e28eeR274

All discount/price recalculation will happen inside `fetch_checkout_prices_if_expired`